pypiserver/_app.py

```diff
@@ -357,25 +406,56 @@ def list_packages():
     return template(tmpl, links=links)
 
 
-@app.route("/packages/:filename#.*#")
+@app.route(f"/packages/<pkg_version:re:(.*-.*)-.*-.*-.*\.whl\.metadata>")
+@auth("download")
+def serve_metadata(pkg_version):
+    pkg, version, _ = pkg_version.split("-", 2)
+    urs = urlsplit(config.fallback_url.rstrip("/"))
+    json_url = f"{urs.scheme}://{urs.hostname}/pypi/{pkg}/json"
+    response = urllib.request.urlopen(json_url)
+    content = json.loads(response.read().decode("utf-8"))
+    metadata_url = [
+        i["url"]
+        for i in content["releases"][version]
+        if i["packagetype"] == "bdist_wheel"
+    ][0]
+    response = urllib.request.urlopen(metadata_url + ".metadata")
+    content = response.read().decode("utf-8")
+    response = HTTPResponse(body=content, status=200)
+    return response
```

Comment:
[curious] Hey @oz123, do I understand this correctly that this endpoint retrieves `metadata` fully based on the `fallback_url`? Do we not need to consider local packages in this case?